backend/drm/util.c

```diff
@@ -170,6 +170,25 @@ const char *conn_get_name(uint32_t type_id) {
 	}
 }
 
+static uint32_t get_fb_for_bo_legacy(struct gbm_bo *bo) {
+	struct gbm_device *gbm = gbm_bo_get_device(bo);
+
+	int fd = gbm_device_get_fd(gbm);
+	uint32_t width = gbm_bo_get_width(bo);
+	uint32_t height = gbm_bo_get_height(bo);
+	uint32_t depth = 24;
```

Comment:
This depends on the format.

Reply:
Per discussion in IRC, we locked this to `GBM_FORMAT_ARGB8888` and changed depth to 32.  We've also validated this is working as expected on a machine where ADDFD2 is not working (due to the driver not supporting it on big-endian).